cmd/mindthegap/push/bundle/bundle.go

```diff
@@ -228,3 +249,69 @@ func pushImages(
 
 	return nil
 }
+
+func pushOCIArtifacts(
+	cfg config.HelmChartsConfig,
+	sourceRegistry, destRegistry string,
+	remoteOpts []remote.Option,
+	out output.Output,
+	prePushFuncs ...prePushFunc,
+) error {
+	// Sort repositories for deterministic ordering.
+	repoNames := cfg.SortedRepositoryNames()
+
+	for _, repoName := range repoNames {
+		repoConfig := cfg.Repositories[repoName]
+
+		// Sort charts for deterministic ordering.
+		chartNames := repoConfig.SortedChartNames()
+
+		for _, chartName := range chartNames {
+			chartVersions := repoConfig.Charts[chartName]
+
+			for _, prePush := range prePushFuncs {
+				if err := prePush("", destRegistry); err != nil {
+					return fmt.Errorf("pre-push func failed: %w", err)
+				}
+			}
+
+			for _, chartVersion := range chartVersions {
+				out.StartOperation(
+					fmt.Sprintf("Copying %s:%s (from bundle) to %s/%s:%s",
+						chartName, chartVersion,
+						destRegistry, chartName, chartVersion,
+					),
+				)
+
+				srcChart := fmt.Sprintf("%s/%s:%s", sourceRegistry, chartName, chartVersion)
+				srcChartRef, err := name.ParseReference(srcChart, name.StrictValidation)
+				if err != nil {
+					out.EndOperation(false)
+					return err
+				}
+				src, err := remote.Image(srcChartRef)
+				if err != nil {
+					out.EndOperation(false)
+					return err
+				}
+
+				destChart := fmt.Sprintf("%s/%s:%s", destRegistry, chartName, chartVersion)
+				fmt.Println(destChart)
```

Comment:
Is this a debugging output? If not should we use `out` for writing this info?

Reply:
Good spot! Removed.